.gitignore

```diff
@@ -12,3 +12,11 @@ web/.doit.db.bak
 web/.doit.db.dat
 web/.doit.db.db
 web/.doit.db.dir
+*~
+.idea
+*#
+#*
```

Comment:
Okay, what do these two do?

Reply:
Files starting or ended with `#` sometimes show up as temporary emacs files, which we should ignore.